### PR TITLE
2023 Digital Infrastructure Insights Fund RFP

### PR DESCRIPTION
**Pull request by BatoolMM**
### Summary

This is the grant proposal we submitted for 2023 Digital Infrastructure Insights Fund RFP
to fund the localisation of the TTW.

### What should a reviewer concentrate their feedback on?

- [ ] Metadata, I haven't added the contributors of the proposal - I can add them and order them alphabetically - what else I should be adding?
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: project_management/proposals/digital-Infrastructure-insights-fund.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+
```
Comment (KirstieJane) [inline project_management/proposals/digital-Infrastructure-insights-fund.md:1]:
```suggestion
```

File: project_management/proposals/digital-Infrastructure-insights-fund.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+# 2023 Digital Infrastructure Insights Fund RFP
+
+
+**Call for Proposals:** [https://fordfoundation.forms.fm/2023-digital-infrastructure-insights-fund-rfp/forms/9724/](https://fordfoundation.forms.fm/2023-digital-infrastructure-insights-fund-rfp/forms/9724/) 
+
+**Deadline:** 1 Oct 2023
+
```
Comment (KirstieJane) [inline project_management/proposals/digital-Infrastructure-insights-fund.md:8]:
```suggestion

Written by (alphabetical order): Batool Almarzouq, Arielle Bennett, Melissa Black, Andrea Sánchez-Tapia, Malvika Sharan, Elisabeth Wadge and Kirstie Whitaker
```

File: project_management/proposals/digital-Infrastructure-insights-fund.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+Our proposal strongly aligns with four of your Fund’s interests:
+
+* Global Shifts & Localization: Demographics and further narratives of maintainership;
+
+* The Human Infrastructure in Digital Infrastructure;
+
+* Sectoral Adoption & Usage of Open Source from Basic Research to Field Work and Operations;
+
+* Social, Material, Environmental (...) Dimensions of Sustainability.
```
Comment (KirstieJane) [inline project_management/proposals/digital-Infrastructure-insights-fund.md:26]:
```suggestion
* Global Shifts & Localization: Demographics and further narratives of maintainership;
* The Human Infrastructure in Digital Infrastructure;
* Sectoral Adoption & Usage of Open Source from Basic Research to Field Work and Operations;
* Social, Material, Environmental (...) Dimensions of Sustainability.
```

**Review by KirstieJane — COMMENTED** [review]
Thank you so much @BatoolMM 

The only comment I have is that some of the links are formatted as links `[link](link)` and others are just the links themselves.

Would it be possible to harmonise? I think in github the raw links are formatted as links, right? So we can drop the `[link](link)` formatting?

A couple of other very small formatting suggestions, and a list of names!

**Comment by BatoolMM** [comment]
The hyperlinks formatting was a disaster - thank you for spotting it- thank you both for the quick review!

**Comment by melibleq** [comment]
Thank you Batool, Malvika, and Kirstie! I think it looks great and agree with your proposal for the authorship metadata @BatoolMM. Not sure what else could be added either 🤔

**Review by melibleq — APPROVED** [review]
Approving the document!

**Comment by malvikasharan** [comment]
Folks I am merging this so that I can share this with Carly. Thanks a ton for this inspiring work. <3

**Pull request merged**